datafusion/core/tests/sql/select.rs

```diff
@@ -250,3 +250,33 @@ async fn test_parameter_invalid_types() -> Result<()> {
 );
     Ok(())
 }
+
+#[tokio::test]
+async fn test_version_function() {
+    let expected_version = format!(
```

Comment:
The best I can come up with was to programatically create the same expected output. 

It would be better to make it with something different, but given it changes with architecture and release this is the best I could come up with